src/Components/Server/src/Circuits/RemoteRenderer.cs

```diff
@@ -179,7 +179,7 @@ private async Task WriteBatchBytesAsync(PendingRender pending)
                     return;
                 }
 
-                Log.BeginUpdateDisplayAsync(_logger, _client.ConnectionId);
+                Log.BeginUpdateDisplayAsync(_logger, _client.ConnectionId, pending.BatchId, pending.Data.Length);
```

Comment:
While you're editing this file, could you update the log message on line 218 to use `LoggerMessage`?

Reply:
There's also some in `RemoteUriHelper` and `ComponentHub`:

* https://github.com/aspnet/AspNetCore/blob/master/src/Components/Server/src/ComponentHub.cs#L76
* https://github.com/aspnet/AspNetCore/blob/master/src/Components/Server/src/ComponentHub.cs#L134
* https://github.com/aspnet/AspNetCore/blob/master/src/Components/Server/src/ComponentHub.cs#L143
* https://github.com/aspnet/AspNetCore/blob/master/src/Components/Server/src/ComponentHub.cs#L152
* https://github.com/aspnet/AspNetCore/blob/master/src/Components/Server/src/Circuits/RemoteUriHelper.cs#L63
* https://github.com/aspnet/AspNetCore/blob/master/src/Components/Server/src/Circuits/RemoteUriHelper.cs#L82
* https://github.com/aspnet/AspNetCore/blob/master/src/Components/Server/src/Circuits/RemoteUriHelper.cs#L89

Reply:
Sure, good plan.